TUM Campus App/PersonDetailTableViewController.swift

```diff
@@ -93,13 +93,20 @@ extension PersonDetailTableViewController {
             }
         }
         let cell = tableView.dequeueReusableCell(withIdentifier: "contact") ?? UITableViewCell()
-        cell.textLabel?.text = contactInfo[indexPath.row].0.rawValue
-        cell.detailTextLabel?.text = contactInfo[indexPath.row].1
+        if indexPath.row < contactInfo.count{
```

Comment:
Please adhere to the rough coding guidelines we have. Space before the `{` and `else` in the same line `}`. For the sake of consitency